x-pack/filebeat/module/rabbitmq/log/ingest/pipeline.json

```diff
@@ -0,0 +1,36 @@
+{
+  "description": "Pipeline for parsing RabbitMQ logs",
+  "processors": [
+    {
+      "grok" : {
+        "field" : "message",
+        "pattern_definitions" : {
+          "GREEDYMULTILINE" : "(.|\n)*",
```

Comment:
This seems useful to other beats. Think it's worth a PR to https://github.com/logstash-plugins/logstash-patterns-core/blob/master/patterns/grok-patterns ?

Reply:
So far we never contributed patterns back to the grok patterns. I kind of like the idea but don't know how grok pattern etc. are versioned and then make it into the products. @jsvd might know more here.

Additional note: The grok patterns you linked above are the ones used by Logstash, I assume there is a different link for Elasticsearch.

Reply:
Re: Additional note
Here's the patterns used by the ingest processor:
https://github.com/elastic/elasticsearch/blob/master/libs/grok/src/main/resources/patterns/grok-patterns